src/renderer/components/cluster-manager/cluster-manager.tsx

```diff
@@ -35,13 +35,31 @@ import { Extensions, extensionsRoute } from "../+extensions";
 import { HotbarMenu } from "../hotbar/hotbar-menu";
 import { EntitySettings, entitySettingsRoute } from "../+entity-settings";
 import { Welcome, welcomeRoute, welcomeURL } from "../+welcome";
+import { TopBar } from "../layout/topbar";
+import { hasLoadedView } from "./lens-views";
+import { navigate } from "../../navigation";
+import { Icon } from "../icon";
+import { MaterialTooltip } from "../material-tooltip/material-tooltip";
+import { getActiveClusterEntity } from "../../api/catalog-entity-registry";
 
 @observer
 export class ClusterManager extends React.Component {
   render() {
+    const cluster = getActiveClusterEntity();
+    const isClusterVisible = cluster?.available && cluster?.ready && hasLoadedView(cluster.id);
```

Comment:
Probably better to extract to helper-function since used in `cluster-view.tsx` already.